vtex/billing-app
# Log installs into Splunk

#### What is the purpose of this pull request?
Log app purchases into Splunk

#### What problem is this solving?
App purchases have not been measured so far

#### How should this be manually tested?
1. Clone/pull this repo/branch and link this app in your workspace
2. Purchase an app such as `compasso.correioslog` by doing `vtex install compasso.correioslog --verbose`
3. If the installation is successful, you can try this query in Splunk
`index=colossus key=log_info sender="vtex.billing@0.5.0" account={{your account}} workspace={{your workspace}}`

A info log of the purchase should appear

#### Types of changes
